app/utils/errors.js

```diff
@@ -1,29 +1,30 @@
 const logger = require('../utils/Logger')
 
 class HttpResponseError extends Error {
-  constructor({ code = 500, message, service }) {
+  constructor({ code = 500, message, req = { path: 'unknown' } }) {
```

Comment:
not sure I understand adding `req` to errors but I might be missing something? `this.path` and `this.request` aren't used anywhere, and since errors are handled as middleware we already have `req`. I can see how we don't really need `service` though, and can just use `req.path`

Reply:
this is what I originally wanted, but after the session with Matt the decision was made to put the entire request, mainly for troubleshooting purposes when it throws